pkg/ingester/ingester_v2_test.go

```diff
@@ -743,3 +744,85 @@ func newIngesterMockWithTSDBStorage(ingesterCfg Config, registerer prometheus.Re
 
 	return ingester, cleanup, nil
 }
+
+func TestIngester_V2Startup(t *testing.T) {
```

Comment:
I would name this test to better explain we're testing the loading of TSDB on startup, so something like `TestIngester_v2LoadTSDBOnStartup`

Reply:
Fixed